moredata/models/merging_data.py

```diff
@@ -0,0 +1,17 @@
+import geopandas
+class MergingData:
+
+    @classmethod
+    def from_geodataframe(self,geodataframe):
+        self.data = geopandas.read_file(geopandas.datasets.get_path(geodataframe))
+        return self.data
```

Comment:
In classmethods we should return only `self`. The same applies to `from_path` method